e2e/fuse-correctness-workload.sh

```
@@ -0,0 +1,610 @@
+stat_nlink() {
+  local file_path="$1"
+  case "$(uname -s)" in
+    Darwin) stat -f %l "$file_path" ;;
+    *) stat -c %h "$file_path" ;;
+  esac
+}
+
+relative_sorted_find() {
+  local root="$1"
+  local type_flag="$2"
+  find "$root" -type "$type_flag" -print | python3 - "$root" <<'PY'
```

Shellcheck (0.11.0)

[error] This redirection overrides piped input. To use both, merge or pass filenames.

(SC2259)